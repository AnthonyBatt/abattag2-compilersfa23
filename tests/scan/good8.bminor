.11
11.11
+11.11
-11.11
+.11
-.11
11.11e11
-11.11e+11

