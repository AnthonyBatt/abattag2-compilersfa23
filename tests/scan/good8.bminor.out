TOKEN_FLOAT_LITERAL       .11
TOKEN_FLOAT_LITERAL       11.11
TOKEN_FLOAT_LITERAL       +11.11
TOKEN_FLOAT_LITERAL       -11.11
TOKEN_FLOAT_LITERAL       +.11
TOKEN_FLOAT_LITERAL       -.11
TOKEN_FLOAT_LITERAL       11.11e11
TOKEN_FLOAT_LITERAL       -11.11e+11
